app/models/course/assessment/skill.rb

```diff
@@ -7,7 +7,7 @@ class Course::Assessment::Skill < ApplicationRecord
   validates :course, presence: true
 
   belongs_to :course, inverse_of: :assessment_skills
-  belongs_to :skill_branch, class_name: Course::Assessment::SkillBranch.name, inverse_of: :skills, optional: true
+  belongs_to :skill_branch, class_name: Course::Assessment::SkillBranch.name, inverse_of: :skills
```

Comment:
@jeremyyap should we do this? it looks like everything is in place to support uncategorized skills (skills that does not belong to any branch). what should happen to the previously uncategorized skills?